README.md

```diff
@@ -1,3 +1,14 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Wikidata Mismatch Finder](#wikidata-mismatch-finder)
+  - [User Guide](#user-guide)
+  - [Administration Guides](#administration-guides)
+  - [Local Development](#local-development)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
```

Comment:
In the past, I intentionally left this one out, since it is sort of a TOC in it's own right, also it's quite short

Reply:
oh ok